src/tiler/server.js

```diff
@@ -36,8 +36,12 @@ var interactivity = {
         drb_streams_v2: 'drb_streams_50',
         nhd_streams_v2: 'nhdflowline',
         municipalities: 'dep_municipalities',
-        urban_areas: 'dep_urban_areas'
+        urban_areas: 'dep_urban_areas',
+        drb_catchment_water_quality_tn: 'drb_catchment_water_quality_tn',
```

Comment:
I'm wondering if a comment is appropriate here.  In other cases this table map actually does map to a db table, where as this maps to an alias, which is referenced in the style sheet.  The actual table is then defined later.  Since it breaks the pattern, it might good to point that out specifically.